src/DssSpell.t.sol

```diff
@@ -2288,7 +2288,10 @@ contract DssSpellTest is DSTest, DSMath {
         checkAuth(true);
     }
 
-    function test_core_unit_budgets() public {
+    function test_core_unit_budgets_distribute() public {
+        // Set done to false in previous spell
+        hevm.store(spellValues.previous_spell, bytes32(uint256(2)), bytes32(0));
```

Comment:
Can we validate after this one that `done` in false?

Reply:
Done.